src/test/java/bio/terra/common/fixtures/Names.java

```diff
@@ -7,7 +7,7 @@ public final class Names {
   private Names() {}
 
   public static String randomizeName(String baseName) {
-    String name = baseName + UUID.randomUUID().toString();
+    String name = baseName + UUID.randomUUID();
     return StringUtils.replaceChars(name, '-', '_');
```

Comment:
There's no need to use `StringUtils` since `String` does this already.
```suggestion
    return name.replace('-', '_');
```